### DISCUSSION TITLE
Adjusting Modulus of Elasticity for existing material

### DISCUSSION THREAD

**Discussion by JesperWB** [Dlubal-Software/RFEM_Python_Client#307] [Q&A] [answered]
Hi,

I'm trying to adjust the E-modulus of an existing material (concrete C30/37) in RFEM6 using the python client. Is this possible? If not, would it be possible to enter a user defined material?

Best regards,
Jesper

**Comment by OndraMichal** [comment, marked as answer]
Hi @JesperWB,
sorry for the delay. Sure, you can. MAterial.UserDefinedMaterial() method was added today to facilitate it. PR #331.
